internal/db/remote/changes/changes.go

```diff
@@ -278,8 +278,7 @@ EOSQL
 }
 
 func cleanup() {
-	utils.DockerRemoveAll()
-	_ = utils.Docker.NetworkRemove(context.Background(), netId)
+	_ = utils.DockerRemoveAll(context.Background(), netId)
```

Comment:
Since there's only 1 method, let's unwrap the cleanup function and call `DockerRemoveAll` in place instead. For eg.

```go
defer utils.DockerRemoveAll(context.Background(), netId)
```